TestClass.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <cassert>  
+
+using namespace std; //use std::cout
+
+#define FILE (__FILE__)
+#define LINE (__LINE__)
+
+// #define LOG_LENGTH (300)
+const int LOG_LENGTH = 300;
+
+void print_log(const char *filename,
```

Comment:
C++ style (you can use `std::string` instead of the `const char *filename`)

you can 
`#include <string>`

then if the parameter is not going to be changed by the function use `const string&` 

Generally, one-liner functions can be avoided (in production environment). For the exercise it is ok.

Reply:
done